### PR TITLE
[tests-only][full-ci]Run ocm tests on ci

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This PR add code to run ocm tests suite on CI

## Related Issue

- Part of https://github.com/owncloud/web/issues/11680

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -127,6 +129,23 @@ config = {
                 "IDP_ACCESS_TOKEN_EXPIRATION": 30,
             },
         },
+        "ocm": {
+            "earlyFail": True,
+            "skip": False,
```
Comment (saw-jan) [inline .drone.star:134]: @ScharfViktor should this be nightly only thing or run on PRs as well?

Comment (ScharfViktor) [reply .drone.star:134]: let's run ocm suite at night and when [full-ci]

File: .drone.star [hunk 2]
```diff
@@ -567,6 +587,9 @@ def e2eTests(ctx):
         if "app-provider" in suite and not "full-ci" in ctx.build.title.lower() and ctx.build.event != "cron":
             continue
 
+        if "ocm" in suite and not "full-ci" in ctx.build.title.lower() and ctx.build.event != "cron":
+            continue
+
```
Comment (saw-jan) [inline .drone.star:592]: ocm pipeline is still running

**Review by saw-jan — CHANGES_REQUESTED** [review]
Please, make ocm pipeline green and then switch it off for PRs

File: .drone.star (hunk 2 above)
Comment (amrita-shrestha) [reply .drone.star:592]: ocm has been triggering because PR has full-ci in title

**Review by saw-jan — APPROVED** [review]

**Pull request merged**